mlir/lib/Conversion/ArithToAMDGPU/ArithToAMDGPU.cpp

```diff
@@ -68,6 +72,15 @@ struct TruncfToFloat16RewritePattern final
 
 } // end namespace
 
+static LogicalResult isSupportedFp8(Type elementType, Chipset chipset) {
```

Comment:
nit: `supportsF8`?

Reply:
I think we've been using Fp8 in this file? But my memory of having written this code ages ago could be hazy

Reply:
The various ops use "fp8" as do the saturate parameters.  I made this guy isSupportedF8().